javaharness/java/arcs/android/demo/BUILD

```diff
@@ -16,9 +16,9 @@ android_binary(
     deps = [
         "//javaharness/java/arcs/android",
         "//javaharness/java/arcs/api:api-android",
-        "@com_google_dagger",
-        "@flogger//jar",
+        "//third_party/java/dagger",
+        "//third_party/java/flogger",
         "@flogger_system_backend//jar",
-        "@javax_inject_source//jar",
+        "//third_party/java/daggerx_inject_source//jar",
```

Comment:
Double slash?

Reply:
BTW, I have a PR to fix the kotlinx dependencies in the same manner, but it won't work until the workspaces are merged